config/routes.rb

```diff
@@ -3,7 +3,7 @@
 
   namespace :api do 
     namespace :v1 do
-      resources :campsites, only: :show
+      resources :campsites, only: [:show, :index]
```

Comment:
Looks like we didn't catch this while fixing our merge conflict -- we'll make sure to test on localhost in the future before we deploy.